### PR TITLE
docs: update readme to latest changes

### PR DESCRIPTION
**Pull request by Pranavchiku**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,11 +48,11 @@ Development, and test the extension. :)
 To package the extension, you can do:
 
 ```bash
-npm install -g vsce
+sudo npm install -g vsce
```
Comment (certik) [inline README.md:51]: That's not a good idea. Never use `sudo` for development.

Let's figure out how to do this without `sudo`.

Comment (certik) [reply README.md:51]: We have to find another way.

Comment (dylon) [reply README.md:51]: There is no need to install it globally, it works well locally:

```bash
npm install vsce
npx vsce package
```

Comment (Pranavchiku) [reply README.md:51]: Yes, this works.

Comment (Pranavchiku) [inline README.md:51]:
```suggestion
npm install --save-dev vsce
```

**Review by certik — APPROVED** [review]
I think that's fine.

**Pull request merged**